src/utils/headers.js

```diff
@@ -94,9 +96,9 @@ const parseHeadersFile = function (filePath) {
       throw new Error('path should come before headers')
     }
 
-    if (line.includes(':')) {
-      const [key = '', value = ''] = line.split(':', 2)
```

Comment:
This is a port of our Ruby code. TIL `limit` in Ruby's split doesn't mean the same thing as `limit` in JavaScript's split.